mlir/test/Pass/crashless-reproducer.mlir

```diff
@@ -0,0 +1,13 @@
+// RUN: mlir-opt %s -pass-pipeline='builtin.module(builtin.module(test-module-pass))' \
+// RUN:   -mlir-pass-pipeline-crash-reproducer=%t \
+// RUN:   -mlir-pass-pipeline-always-generate-reproducer=true -verify-diagnostics
```

Comment:
This test is not working at the moment, working on it. I have something working manually on my local machine.